controllers/index.js

```diff
@@ -1,9 +1,15 @@
 const router = require('express').Router();
 
 const apiRoutes = require('./api');
-const homeRoutes = require('./homeRoutes');
+const homeRoutes = require('./homeRoutes.js');
+const profileRoutes = require('./profileRoutes.js');
 
 router.use('/', homeRoutes);
 router.use('/api', apiRoutes);
+router.use('/profile', profileRoutes);
```

Comment:
This should be changed to userRoutes

Just saw the profileRoutes you created below so maybe disregard this.